src/web/docusaurus/docusaurus.config.js

```diff
@@ -41,7 +41,7 @@ const config = async () => {
     ],
     i18n: {
       defaultLocale: 'en',
-      locales: ['en', 'vi'],
+      locales: ['en', 'vi', 'bg'],
```

Comment:
I think this is part of a different PR, along with the changes in `overview.md`.

Reply:
Yes this is from the wrong PR. I have removed the changes.